src/pages/resultsView/mutualExclusivity/MutualExclusivityUtil.spec.tsx

```diff
@@ -423,9 +423,9 @@ describe("MutualExclusivityUtil", () => {
             assert.equal(cells.at(6).html(), "<td><span>&gt;3</span></td>");
             assert.equal(cells.at(7).html(), "<td><span>0.004</span></td>");
             assert.equal(cells.at(8).html(), "<td><b><span>0.024</span></b></td>");
-            assert.equal(cells.at(9).html().replace(/<!--[^>]*-->/g, ""), "<td><table><tr><td>Co-occurrence</td></tr>" +
-                "<tr><td><span class=\"badge\" style=\"background-color: rgb(88, 172, 250);\">Significant</span></td></tr></table></td>");
-            assert.equal(cells.at(21).html().replace(/<!--[^>]*-->/g, ""), "<td><table><tr><td>Co-occurrence</td></tr></table></td>");
+            assert.equal(cells.at(9).html().replace(/<!--[^>]*-->/g, ""), "<td><div class=\"styles-module__Tendency__2Y46X\">" + 
```

Comment:
I wonder if this will work because that hash may or may not change from build to build.  I guess we'll find out!

Reply:
By has I mean "styles-module__Tendency__2Y46X\"